cg/meta/workflow/fluffy.py

```diff
@@ -92,14 +92,17 @@ def link_fastq_files(self, case_id: str, dry_run: bool = False) -> None:
         Links fastq files from Housekeeper to case working directory
         """
         case_obj: models.Family = self.status_db.family(case_id)
+        latest_flow_cell = self.status_db.get_latest_flow_cell_on_case(family_id=case_id)
         workdir_path = self.get_workdir_path(case_id=case_id)
         if workdir_path.exists() and not dry_run:
             LOG.info("Fastq directory exists, removing and re-linking files!")
             shutil.rmtree(workdir_path, ignore_errors=True)
         workdir_path.mkdir(parents=True, exist_ok=True)
         for family_sample in case_obj.links:
             sample_id = family_sample.sample.internal_id
-            files: Query = self.housekeeper_api.files(bundle=sample_id, tags=["fastq"])
+            files: Query = self.housekeeper_api.files(
+                bundle=sample_id, tags=["fastq", latest_flow_cell.name]
```

Comment:
this will break eventually, for example if we try to re-deliver old fastq what lost the tag due to spring

Reply:
will adress the fact that fastqs are not stored with fc-tag after decompression in another PR